lib/openshift/cluster_logging.rb

```diff
@@ -124,19 +124,19 @@ def wait_until_kibana_is_ready(user: nil, quiet: false, timeout: 10*60)
       end
     end
 
-    private def collection_spec(user: nil, quiet: false, cached: false)
+    def collection_spec(user: nil, quiet: false, cached: false)
```

Comment:
@QiaolingTang any reason you changed these methods from `private`?

Reply:
@pruan-rht In clusterlogging, the collection_spec, log_store_spec, curation_spec and visualization_spec are optional. In some scenarios we may not define all of them when creating clusterlogging/instance. I want to use these methods to check if these specs are defined or not. An example: https://github.com/openshift/verification-tests/pull/1935/files#diff-25b7b9fb37c470ca782e7381a157ec7648e98a1a80f61f2cce948bba99bf676cR667

Reply:
/lgtm